src/test/java/com/adobe/commerce/cif/model/PojoTest.java

```diff
@@ -59,13 +59,17 @@ private SetterGetter(Method setter, Method getter) {
     @Test
     public void testSettersAndGetters() throws Exception {
         Set<ClassInfo> classes = ClassPath.from(this.getClass().getClassLoader()).getTopLevelClassesRecursive(MODEL_PACKAGE);
-        for (Class<?> clazz : classes.stream().map(c -> c.load()).collect(Collectors.toList())) {
-            LOGGER.debug("Checking setter/getter for " + clazz.getName());
```

Comment:
Why did you remove the debug? I think it's quite useful to make sure that for example we test all the expected classes.

Reply:
If it's useful then maybe we should promote the message to `info` level?

Reply:
Should stay at `debug`, otherwise it's creating too much output, but I activated that many times when developing the test to make sure it really tested everything.